src/Admin/BaseBlockAdmin.php

```diff
@@ -84,7 +84,11 @@ protected function alterObject(object $object): void
 
     protected function alterNewInstance(object $object): void
     {
-        $object->setType($this->getPersistentParameter('type'));
+        $type = $this->getPersistentParameter('type');
+
+        if (null !== $type) {
+            $object->setType($type);
```

Comment:
We need to only do the setType if we are sure is a string, because of the signature:

```
    public function setType(string $type): void
    {
        $this->type = $type;
    }
```

TBH this is happening a lot, considering that model is meant to be an entity, IMHO it would make much more sense to treat it like that and make all fields that starts with a null value accept a null one (Like we are doing on each bundle that has an entity).

But I guess we can deal with that in another PR, wdyt  @VincentLanglet ?